nginx/proxy_dev.conf

```diff
@@ -2,18 +2,10 @@ upstream mapproxy {
   server mapproxy:8080;
 }
```

Comment:
So far we've been using `proxy_dev.conf` in both local and `lipas-dev.cc.jyu.fi` environments. It might be a good time to have a separate more local friendly config. I propose we leave `proxy_dev` for the `lipas-dev.cc.jyu.fi` and create a new `proxy_local.conf`. But feel free to do this the way you see best.